conf/airframes/examples/quadrotor_lisa_m_2_pwm_spektrum.xml

```diff
@@ -57,24 +57,16 @@
   </commands>
 
   <section name="MIXING" prefix="MOTOR_MIXING_">
-    <define name="TRIM_ROLL" value="0"/>
-    <define name="TRIM_PITCH" value="0"/>
-    <define name="TRIM_YAW" value="0"/>
-    <define name="NB_MOTOR" value="4"/>
-    <define name="SCALE" value="256"/>
     <!-- front/back turning CW, right/left CCW -->
-    <define name="ROLL_COEF"   value="{    0,    0, -256,  256 }"/>
-    <define name="PITCH_COEF"  value="{  256, -256,    0,    0 }"/>
-    <define name="YAW_COEF"    value="{ -256, -256,  256,  256 }"/>
-    <define name="THRUST_COEF" value="{  256,  256,  256,  256 }"/>
+    <define name="TYPE" value="QUAD_PLUS"/>
```

Comment:
Integrated the feature into my [hooperfly_motor_mixing] branch to exercise the HEXA_X code. Identified an issue with the MOTOR_MIXING_ROLL_COEF rotation direction in sw/airborne/subsystems/actuators/motor_mixing_types.h on line 82:

#define MOTOR_MIXING_ROLL_COEF   {  181, -181, -256,  181, -181,  256 }
should be
#define MOTOR_MIXING_ROLL_COEF   {  181, -181, -256,  -181, 181,  256 }

Will report back after field testing of the HEXA_X default settings as defined in sw/airborne/subsystems/actuators/motor_mixing_types.h